src/db/tables/onerep_profiles.ts

```diff
@@ -12,18 +12,33 @@ export async function setProfileDetails(
   onerepProfileId: number,
   profileData: CreateProfileRequest,
 ) {
+  const {
+    addresses,
```

Comment:
I think the destructuring here assumes that addresses is always an array with at least one element. If addresses is `empty or undefined`, the code might throw an error.

Reply:
As per the type [CreateProfileRequest](https://github.com/mozilla/blurts-server/blob/mntor-3144/src/app/functions/server/onerep.ts#L33) an array with at least one element is what we expect as an argument for `addresses`.